security.php

```diff
@@ -54,56 +57,202 @@ function wpcom_vip_is_restricted_username( $username ) {
 		|| WPCOM_VIP_MACHINE_USER_EMAIL === $username;
 }
 
+/**
+ * PRIVATE: This function is not intended to be used outside of this file and is subject to change without notice.
+ *
+ * Returns the defualts for limits.
+ */
+function _vip_get_auth_count_limit_defaults() {
+	if ( Context::is_fedramp() ) {
+		return [
+			'ip_username_login'          => 3,
+			'ip_login'                   => 5,
+			'username_login'             => 10,
+			'ip_username_password_reset' => 3,
+			'ip_password_reset'          => 3,
+			'username_password_reset'    => 15,
+
+			'ip_username_window'         => MINUTE_IN_SECONDS * 15,
+			'ip_window'                  => MINUTE_IN_SECONDS * 15,
+			'username_window'            => MINUTE_IN_SECONDS * 15,
+
+			'ip_username_lockout'        => MINUTE_IN_SECONDS * 30,
+			'ip_lockout'                 => MINUTE_IN_SECONDS * 30,
+			'username_lockout'           => MINUTE_IN_SECONDS * 30,
+		];
+	}
+
+
+	return [
+		'ip_username_login'          => 5,
+		'ip_login'                   => 50,
+		'username_login'             => 25,
+		'ip_username_password_reset' => 3,
+		'ip_password_reset'          => 3,
+		'username_password_reset'    => 15,
+
+		'ip_username_window'         => MINUTE_IN_SECONDS * 5,
+		'ip_window'                  => MINUTE_IN_SECONDS * 60,
+		'username_window'            => MINUTE_IN_SECONDS * 25,
+
+		'ip_username_lockout'        => MINUTE_IN_SECONDS * 5,
+		'ip_lockout'                 => MINUTE_IN_SECONDS * 60,
+		'username_lockout'           => MINUTE_IN_SECONDS * 25,
+	];
+}
+
+/**
+ * PRIVATE: This function is not intended to be used outside of this file and is subject to change without notice.
+ *
+ * Returns the cache keys used for login and forgotten passoword rate limiting.
+ *
+ * @param string $raw_username The username to use for the cache keys.
+ * @return array Associative array of cache keys.
```

Comment:
```suggestion
 * Return the cache keys used for login and forgotten password rate limiting.
 *
 * @internal
 *
 * @param string $raw_username The username to use for the cache keys.
 * @return array Associative array of cache keys.
```

Any particular reason this needs to be considered private/internal?